src/lightning/pytorch/utilities/model_summary/model_summary.py

```diff
@@ -292,8 +301,22 @@ def _get_summary_data(self) -> List[Tuple[str, List[str]]]:
             arrays.append(("In sizes", [str(x) for x in self.in_sizes]))
             arrays.append(("Out sizes", [str(x) for x in self.out_sizes]))
 
+        total_leftover_params = self.total_parameters - self.total_layer_params
+        if total_leftover_params > 0:
+            self._add_leftover_params_to_summary(arrays, total_leftover_params)
+
         return arrays
 
+    def _add_leftover_params_to_summary(self, arrays: List[Tuple[str, List[str]]], total_leftover_params: int):
+        """Add summary of params not associated with module or layer to model summary."""
+        layer_summaries = dict(arrays)
+        layer_summaries[" "].append(str(len(self._layer_summary) + 1))
+        layer_summaries["Name"].append(LEFTOVER_PARAMS_NAME)
+        layer_summaries["Type"].append(NOT_APPLICABLE)
+        layer_summaries["Params"].append(get_human_readable_count(total_leftover_params))
+        layer_summaries["In sizes"].append(NOT_APPLICABLE)
+        layer_summaries["Out sizes"].append(NOT_APPLICABLE)
```

Comment:
In and out sizes are optional, these columns may not exist, and would result in a key error.